src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,179 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import tasks.Task;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Todo;
+
+import dukeException.DukeException;
+import dukeException.DukeIOBException;
+
+public class Duke {
+
+    public Duke() {
+
+    }
+
+    /*
+    Main function that takes user input and interpets how to store and what to do with it
+     */
+    public static void main(String[] args) {
+
+        ArrayList<String> userInputs = new ArrayList<>();
+        ArrayList<Task> tasks = new ArrayList<>();
+        Scanner scan = new Scanner(System.in);
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
```

Comment:
I notice that you have a tendency to use magic strings and numbers. Do try to avoid that where possible, and instead use constants to set your strings/numbers.